lib/common.js

```diff
@@ -327,43 +327,77 @@ function matchStringOrRegexp(target, pattern) {
   return pattern instanceof RegExp  ? str.match(pattern) : str === String(pattern);
 }
 
-// return [newKey, newValue]
-function formatQueryValue(key, value, options) {
+function formatQueryParams(queryParams) {
+  var formattedQueryParams = {}
+  for (var q in queryParams) {
+    if (!_.isUndefined(formattedQueryParams[q])) {
```

Comment:
I'm not sure I understand, do you mean multi-value query params were supported and now they are not? I don't see how this would be possible, since we were also not setting a value already in the queries before:
```
for (var q in queries) {
    if (_.isUndefined(this.queries[q])) {
        var value = queries[q];
        var formatedPair = common.formatQueryValue(q, value, this.scope.scopeOptions);
        this.queries[formatedPair[0]] = formatedPair[1];
    }
}
```